src/components/05_pages/NodeForm/index.js

```diff
@@ -41,44 +42,18 @@ class NodeForm extends React.Component {
       }),
       PropTypes.bool,
     ]),
+    onChange: PropTypes.func,
   };
 
   static defaultProps = {
     schema: false,
     uiSchema: false,
+    onChange: () => {},
```

Comment:
Note: Due to storing content in redux typing seemed to be a little bit slow. Maybe that's just me?